src/main/java/uk/gov/ch/exception/ManagingOfficerCountNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.ch.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class ManagingOfficerCountNotFoundException extends Exception {
```

Comment:
Would it sound better to just have ManagingOfficerNotFoundException?

Reply:
its declared as a count as it is a list and is done like this in other places too but i do agree its not very catchy 😄